docker-php-pecl-configure

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+set -e
+
+# prefer user supplied CFLAGS, but default to our PHP_CFLAGS
+: ${CFLAGS:=$PHP_CFLAGS}
+: ${CPPFLAGS:=$PHP_CPPFLAGS}
+: ${LDFLAGS:=$PHP_LDFLAGS}
+export CFLAGS CPPFLAGS LDFLAGS
+
+usage() {
+	echo "usage: $0 ext-name [configure flags]"
+	echo "   ie: $0 apcu --enable-apcu-spinlocks"
+	echo "       $0 -j5 apcu redis grpc protobuf"
+	echo
+	echo 'if custom ./configure arguments are necessary, see docker-php-pecl-configure'
```

Comment:
This refers to itself. Is that correct?

Reply:
Thanks! This is redundant and has been removed.
Error caused by copy & paste.